modules/image-loading-optimization/module.php

```diff
@@ -0,0 +1,264 @@
+<?php
+namespace Elementor\Modules\ImageLoadingOptimization;
+
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+	private $min_priority_img_pixels = 50000;
+
+	public function get_name() {
+		return 'image-loading-optimization';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'wp_get_attachment_image_attributes', [ $this, 'remove_get_attachment_loading_attributes' ], 10, 3 );
```

Comment:
I was aware of the side effect and I usually don't prefer calling hooks in constructor, but the way dynamic loading of the modules are written here, it was the only option to call hooks. Also, since this was something they have followed for all the modules I didn't spend much time on this part.

I have fix `action` to `filter`

Reply:
Assumed you were probably aware, but it wasn't something I remember us discussing before, so figured I'd mention it. Thanks!

Reply:
It's good that you raised it. There are many improvements that can be made in elementor beyond the scope of our current work. Something that we can note and pickup later.